### PR TITLE
Remove Webpack and reorganize codebase structure

### PR DESCRIPTION
**Pull request by tfuxu**
This removes Webpack dependency and simplifies website structure. 
Removing Webpack have increased performance scores for mobile from 71 to **95** and for desktop from 76 to **100**.

Here's a PageSpeed analysis for my test instance: https://pagespeed.web.dev/analysis/https-tfuxu-github-io-python-pillow-github-io/xevcdhxuhu

PS. I'll fix remaining issues with accessibility in the another PR

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,2 +1,56 @@
...
 ==============
+
+This website is live at `python-pillow.org <https://python-pillow.org>`_
+
+Website structure:
+------------------
+
+.. code-block:: text
+
+   ├── assets
```
Comment (hugovk) [inline README.rst:11]: This directory listing feels redundant and is likely to go out of sync. Can we remove it?

Comment (tfuxu) [reply README.rst:11]: Sure, I'll remove it

File: index.html [hunk 2]
```diff
@@ -8,16 +8,24 @@
         <meta name="author"
               content="Fredrik Lundh, Jeffrey A. Clark (Alex), Pillow Contributors">
 
+        <link rel="stylesheet" href="assets/css/pillow.css">
+
+        <link href="assets/fontawesome/css/fontawesome.css" rel="stylesheet">
+        <link href="assets/fontawesome/css/brands.css" rel="stylesheet">
+        <link href="assets/fontawesome/css/solid.css" rel="stylesheet">
```
Comment (hugovk) [inline index.html:15]: Shall we use the minimised ones? 
```suggestion
        <link href="assets/fontawesome/css/fontawesome.min.css" rel="stylesheet">
        <link href="assets/fontawesome/css/brands.min.css" rel="stylesheet">
        <link href="assets/fontawesome/css/solid.min.css" rel="stylesheet">
```

**Comment by aclark4life** [comment]
Is this a follow up to #35 ?

**Comment by tfuxu** [comment]
Yes, this is a follow up

**Comment by aclark4life** [comment]
Thanks!

**Comment by aclark4life** [comment]
Looking good, thanks again @tfuxu 

Screenshot 2023-09-30 at 11 31 22 AM

**Pull request merged**